src/components/Map.vue

```diff
@@ -1196,16 +1216,22 @@ export default {
         }
       }
     },
+    getFwwLayer() {
+      this.fwwLayer = esri.dynamicMapLayer({
+        url: "https://idpgis.ncep.noaa.gov/arcgis/rest/services/NWS_Forecasts_Guidance_Warnings/watch_warn_adv/MapServer",
+        layers: [0, 1],
+        format: "image/png",
+      });
+      this.fwwLayer.addTo(this.map);
+    },
     getRadarLayer() {
       this.radarLayer = esri.dynamicMapLayer({
         url: "https://idpgis.ncep.noaa.gov/arcgis/rest/services/NWS_Observations/radar_base_reflectivity/MapServer",
         // 0: NFHL Availability, 3: FIRM Panels, 14: Cross Sections, 27: Flood Hazard Boundaries, 28: Flood Hazard Zones
         layers: [3],
         format: "image/png",
```

Comment:
```suggestion
        f: "image/png",
```

same here too

Reply:
this is exactly what needed to happen and u are a genius